### DISCUSSION TITLE
Compilation problems

### DISCUSSION THREAD

**Discussion by serzhiio** [hecrj/iced#1501] [Q&A] [answered]
I've created new project and trying to copy one of samples. I discovered that my project compiles only when i refer to downloaded iced dep like `iced={path="../.."}`, but if i put `iced={version="0.4"}` i receive a lot of errors 
Whats wrong with me? :)

**Comment by serzhiio** [comment]
![image](https://user-images.githubusercontent.com/10449591/199234238-b7335c30-a33f-4bc5-8f93-9d34c251e0fc.png)

**Comment by serzhiio** [comment, marked as answer]
I figured out that only this is working `iced = {git="https://github.com/iced-rs/iced" }`

**Reply by 13r0ck** [reply]
Yep the git repo is the master branch, if you want to use iced version 0.4 you will need to check out the 0.4 branch's examples https://github.com/iced-rs/iced/tree/0.4

**Comment by arjunair86** [comment]
All the examples are using functions from `helpers.rs`, which apparently is not present in release version.
